cosmoz-grouped-list.js

```diff
@@ -57,7 +57,8 @@
 
 		observers: [
 			'_dataChanged(data.*)',
-			'_scrollTargetChanged(scrollTarget, _isAttached)'
+			'_scrollTargetChanged(scrollTarget, _isAttached)',
+			'_selectedItemsChanged(selectedItems.*)'
```

Comment:
This will trigger `_selectedItemsChanged` twice, one for `selectedItems.splices` and one for `selectedItems.length`.
Observing  `selectedItems.splices` should be enough.